src/accelerate/utils/dataclasses.py

```diff
@@ -324,6 +326,7 @@ class FP8RecipeKwargs(KwargsHandler):
     """
 
     backend: Backend = "MSAMP"
+    use_during_eval: bool = False
```

Comment:
Strictly speaking, this change is not 100% backwards compatible, right? E.g. some existing eval code using FP8 could now OOM when previously it didn't. It's probably still fine to change it, as this is an edge case, but I wanted to flag it.

Reply:
It could but the old method was "wrong". We can make a note about it though